plugins/modules/aci_bd.py

```diff
@@ -449,6 +457,7 @@ def main():
                 unicastRoute=enable_routing,
                 unkMacUcastAct=l2_unknown_unicast,
                 unkMcastAct=l3_unknown_multicast,
+                v6unkMcastAct=ipv6_l3_unknown_multicast,
```

Comment:
@tvarohohlavy 
We need to add a condition when including v6unkMcastAct in the payload since it's only applicable to newer APIC versions. Let me check how this can be done.

Reply:
Using this module as an example: https://github.com/CiscoDevNet/ansible-aci/blob/c4a9cc70eecf088cc7c8465968c9186036036da8/plugins/modules/aci_l3out_static_routes.py#L344

We can do -
Remove line 460 and add the lines below after payload is configured:

if ipv6_l3_unknown_multicast is not None:
    class_config['v6unkMcastAct'] = ipv6_l3_unknown_multicast

Reply:
For the test we can add something like this -

```
- name: Query system information
  aci_system:
    <<: *aci_info
    id: 1
    state: query
  register: version

- name: create another bd - check more params
  cisco.aci.aci_bd:
    <<: *aci_bd_present
    bd: anstest2
    bd_type: ethernet
    endpoint_move_detect: default
    ip_learning: "no"
    l2_unknown_unicast: flood
    l3_unknown_multicast: opt-flood
    ipv6_l3_unknown_multicast: opt-flood
    multi_dest: drop
    enable_routing: "no"
    arp_flooding: "yes"
    route_profile_l3out: ansible_l3out
    route_profile: ansible_l3out_route
  register: bd_present_2
  when: version.current.0.topSystem.attributes.version is version('4.2', '>=')

- name: Check for ipv6_l3_unknown_multicast for select versions
  assert:
    that:
       - bd_present_2.sent.fvBD.attributes.v6unkMcastAct == 'opt-flood'
   when: version.current.0.topSystem.attributes.version is version('4.2', '>=')